infra/dev/laravel/Dockerfile

```diff
@@ -3,17 +3,35 @@ FROM php:7.4-fpm
 # unlock php deb package
 RUN rm /etc/apt/preferences.d/no-debian-php
 
-ARG USER_ID
-ARG GROUP_ID
+# Raw code till i haven't found a way to pass .env file values
+ENV USER_ID 1000
+ENV GROUP_ID 1000
 
 RUN groupadd -f $GROUP_ID
 RUN useradd -u $USER_ID -g $GROUP_ID user
 
+RUN mkdir /home/user
```

Comment:
Why do we need to create a home directory?

Reply:
Because it doesn't exist, needed by npm (error if not exist)